From: Colin Walters <walters@verbum.org>
Date: Tue, 8 Aug 2017 17:26:23 -0400
Subject: README.md: Delete cat logo picture (not DFSG compliant)

The cat is cute, but let's not hinder adoption anywhere based on this. If
someone cares, we could probably project the Internet emergency logo into the
sky, requesting a cat picture with bubblewrap that is *also* DFSG compliant.

[In the Debian package the picture was already removed from the orig
tarball, so this patch only removes the reference to it. -smcv]

Bug: https://github.com/projectatomic/bubblewrap/issues/204
Bug: https://github.com/projectatomic/bubblewrap/pull/206
Approved by: alexlarsson
Origin: upstream, 0.1.9, commit:b9c564dbe0c41368fd5f40e8bf8100e733f854d0
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=876980
---
 README.md | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/README.md b/README.md
index d8edd5f..f27c4ac 100644
--- a/README.md
+++ b/README.md
@@ -173,7 +173,3 @@ Whats with the name ?!
 The name bubblewrap was chosen to convey that this
 tool runs as the parent of the application (so wraps it in some sense) and creates
 a protective layer (the sandbox) around it.
-
-![](bubblewrap.jpg)
-
-(Bubblewrap cat by [dancing_stupidity](https://www.flickr.com/photos/27549668@N03/))
